README.md

```diff
@@ -10,6 +10,14 @@ Konveyor Forklift controller.
 
 For the build, the forklift uses [Bazel](https://bazel.build/).
 
+Install the following system components:
+
+- bazel >= 5
+- docker
```

Comment:
actually not sure about docker, it is required for CI. but now sure if bazel needs it.

Reply:
It is not necessary for build. It is only needed if you want to run the final container.

Reply:
it shouldn't be needed for CI as well but maybe [some tweaks are required](https://github.com/kubevirt/kubevirtci/blob/main/PODMAN.md)

Reply:
using podman-docker i didn't try those tweaks. but i had an error about insecure connection to the localhost, when editing the relevant configuration to allow insecure, it still didn't work. with docker it worked out of the box for me. in any case, this is unrelated to the bazel.